eternalcore-core/src/main/java/com/eternalcode/core/feature/burn/BurnCommand.java

```diff
@@ -0,0 +1,81 @@
+package com.eternalcode.core.feature.burn;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.entity.Player;
+
+import java.util.Optional;
+
+@Command(name = "burn", aliases = {"fire"})
```

Comment:
I think the `fire` alias is quite specific to the `burn` command and might not be intuitive for all users. It could be interpreted as a separate or more general command. Unless there's a strong use case for it, I’d consider removing it or picking a more descriptive alternative like `ignite`.